File: 2023-01/Orientação obj/aula02_classe_objeto_construtor_metodos/Livro.cs


namespace aula02_classe_objeto_construtor_metodos
{
    public class Livro
    {
        public int Id {get;private set;}
       
        public string Isbn { get; set; }
         public string  Titulo { get; set; }
        public double Preco { get; set; }

        public Livro(int id, string isbn, string titulo, double preco=0){
            this.Id=id;
            this.Isbn=isbn;
            this.Titulo=titulo;
            this.Preco=preco;

        }

        

       

    }
}

File: 2023-01/Lab. Programação/aula10_ativs/aula10_ativ2/src/Livro.java
public class Livro {

    String titulo;
    String autor;
    double preco;

    public Livro(String titulo, String autor, double preco)
    {
        this.titulo=titulo;
        this.autor=autor;
        this.preco=preco;
    }

    public Livro(String autor, String titulo)
    {
        this.titulo=titulo;
        this.autor=autor;
        this.preco=500.00;
    }
}
